### PR TITLE
Fix test-runner build and update ko version

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

- Update test-runner tooling and fix build
- Update ko to 0.14.1 in all images…

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Review by savitaashture — APPROVED** [review]

**Comment by vdemeester** [comment]
cc @afrittoli

File: tekton/images/test-runner/Dockerfile [hunk 1]
```diff
@@ -54,8 +57,9 @@ RUN apt update && apt install -y --no-install-recommends \
     zlib1g-dev \
     && rm -rf /var/lib/apt/lists/* \
     && update-alternatives --install /usr/bin/python python /usr/bin/python3 1 \
-    && update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 1 \
-    && python -m pip install --upgrade pip setuptools wheel
+    && update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 1
+    # \
+    # && python -m pip install --upgrade pip setuptools wheel
```
Comment (afrittoli) [inline tekton/images/test-runner/Dockerfile:62]: Why this is here as a comment? This is covered now by `python3-setuptools` right?

**Review by afrittoli — COMMENTED** [review]
Thanks!
Hopefully the release will work fine with this version! :)
/lgtm

File: tekton/images/test-runner/Dockerfile (hunk 1 above)
Comment (vdemeester) [reply tekton/images/test-runner/Dockerfile:62]: Yes, my bad, I forgot to remove that comment 😓

**Pull request merged**